relayer/pkg/starknet/utils.go

```diff
@@ -87,26 +87,14 @@ func DecodeFelts(felts []*big.Int) ([]byte, error) {
 	return data, nil
 }
 
-// SignedBigToFelt wraps negative values correctly into felt
-func SignedBigToFelt(num *big.Int) *big.Int {
-	return new(big.Int).Mod(num, caigotypes.MaxFelt.Big())
+func FeltToUnsignedBig(felt *caigotypes.Felt) (num *big.Int) {
```

Comment:
this replaces  FeltToSignedBig. We can remove this since it's a one-liner but I feel it makes it understandable that the results will be unsigned.

Note there's no possibility that FeltToUnsignedBig will return a negative big.Int